api/WorkerNavigator.json

```diff
@@ -244,6 +244,54 @@
           }
         }
       },
+      "deviceMemory": {
+        "__compat": {
+          "spec_url": "https://w3c.github.io/device-memory/#sec-device-memory-js-api",
+          "support": {
+            "chrome": {
+              "version_added": "65"
```

Comment:
This doesn't match api.Navigator.deviceMemory, but I've confirmed that shipped in 63 and worker support came in 65.